Neos.Eel/Classes/Utility.php

```diff
@@ -17,6 +17,19 @@
  */
 class Utility
 {
+    /**
+     * Check if expression is an Eel expression.
+     *
+     * @param string $expression
+     * @param array $matches
+     *
+     * @return bool
+     */
+    public static function isEelExpression($expression, &$matches = null)
+    {
+        return preg_match(Package::EelExpressionRecognizer, $expression, $matches);
```

Comment:
Please lets make this sth like
ˋˋˋ
return preg_match(Package::EelExpressionRecognizer, $expression) === 1);
ˋˋˋ
and get rid of the $matches param